.travis.yml

```diff
@@ -7,8 +7,12 @@ sudo: required
 services:
   - docker
 
+before_script:
+  - go get github.com/chespinoza/goliscan
```

Comment:
Could we move the "go get ..." to a make task e.g. get-deps (or something better with a better name) and then call the make task from the travis file?

It would be useful to be able to install this locally.